tests/integration/kubernetes/gha-run.sh

```diff
@@ -236,6 +239,76 @@ function cleanup() {
 	kubectl delete -f "${tools_dir}/packaging/kata-deploy/kata-rbac/base/kata-rbac.yaml"
 }
 
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
+
+	case ${SNAPSHOTTER} in
+		nydus) deploy_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+
+	echo "::endgroup::"
+}
+
+function deploy_nydus_snapshotter() {
+    echo "deploy_nydus_snapshotter"
+	NODE_NAME=${NODE_NAME:-$(hostname | tr A-Z a-z)} 
```

Comment:
When I try the `deploy_nydus_snapshotter` function on a kcli cluster, the `NODE_NAME` need be set, else it will report follow error:
```
...
daemonset.apps/nydus-snapshotter created
Error from server (NotFound): nodes "liudali-s390x-libvirt" not found
```

Reply:
Fixed.